src/js/containers/Transfer/PostTransfer.js

```diff
@@ -32,24 +39,44 @@ export default class PostTransfer extends React.Component {
     if (Object.keys(this.props.transfer.errors.sourceAmount).length !== 0) {
       return
     }
+    
     if (Object.keys(this.props.transfer.errors.destAddress).length !== 0) {
       return
     }
+  
+    this.setState({isValidating: true});
+    
+    const isTransferValid = await this.validateTransfer();
+  
+    this.setState({isValidating: false});
 
-
-    if (this.validateTransfer()) {
+    if (isTransferValid) {
       var transferPath = [constants.TRANSFER_CONFIG.transferPath.confirm, constants.TRANSFER_CONFIG.transferPath.broadcast]
       this.props.dispatch(transferActions.updateTransferPath(transferPath, 0))
     }
-
   }
 
-  validateTransfer = () => {
+  async validateTransfer() {
     var check = true
     var checkNumber = true
-    if (validators.verifyAccount(this.props.transfer.destAddress.trim()) !== null) {
+    const destAddress = this.props.destAddress.trim();
+  
+    this.props.dispatch(transferActions.setDestEthNameAndAddress(destAddress, ''));
+    
+    if (!destAddress) {
       this.props.dispatch(transferActions.throwErrorDestAddress(constants.TRANSFER_CONFIG.addressErrors.input, this.props.translate("error.dest_address")))
-      check = false
+      return false;
+    }
+    
+    if (validators.verifyAccount(destAddress) !== null) {
+      let { resolvedAddress, isErrorResolvingAddress } = await this.resolveEthNameToAddress(destAddress);
```

Comment:
It should be change to 
let resolvedAddress = await this.resolveEthNameToAddress(destAddress);
if (! resolvedAddress) {
}

Reply:
Done